cla_public/static-src/javascripts/modules/textarea-character-count.js

```diff
@@ -3,11 +3,17 @@
   var LOW_CHAR_COUNT = 80;
 
   if (GOVUK.getCookie("locale") == "cy_GB") {
+    var youHave = '';
```

Comment:
No welsh translation for you have?

Reply:
Not yet.  Since GDS do not prescribe Welsh wording, I'm content with the existing Welsh until we get a translation.  Without this change the existing translation is still valid.